### PR TITLE
PublicDashboards: Title logo and footer redesign

### LINKED ISSUE #62370: Task: Change pubdash footer and add title logo
Now we will always have the pubdash title after a conversation with Torkel. So, does it make sense to put the Grafana footer **powered by Grafana** next to the title? It was a Torkel's suggestion and would be good to have a discussion with the UX team and see if it makes sense and it's prettier

Screenshot 2023-03-14 at 14 38 23

### Solution
The designs are below:

**Desktop:**
PubdashLogo-Desktop

**Tablet:**
PubdashLogo-Tablet

**Mobile:**
PubdashLogo-Mobile

[Figma Link](https://www.figma.com/file/sPfx0P9xsSRVwCEBskyb1O/Public-Dashboards---Email?type=design&node-id=0-1&mode=design)
```[tasklist]
### Tasks
- [x] Add new config key to hide the Grafana logo when enterprise
- [x] Add new config key to public dashboard documentation: https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/configure-custom-branding/
- [x] Add title logo
- [x] Modify footer
- [x] Add referrer to grafana.com
- [ ] Add enterprise doucumentation: https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/enterprise-configuration/
```

### PR DESCRIPTION
**Pull request by juanicabanas**
**What is this feature?**

It adds Grafana logo next to the title and also makes a UI modification in the footer

**Why do we need this feature?**

More adoption

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana/issues/62370

**Special notes for your reviewer:**
[Related Enterprise PR](https://github.com/grafana/grafana-enterprise/pull/5809)

| Light mode  | Dark mode |
| ------------- | ------------- |
| image | image  |

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by chri2547 — COMMENTED** [review]
Docs look fine! Thank you for the contribution!

File: pkg/api/dtos/frontend_settings.go [hunk 1]
```diff
@@ -93,27 +93,27 @@ type FrontendSettingsFooterConfigItemDTO struct {
...
+	LoginTitle         string                                    `json:"loginTitle"`
+	AppTitle           *string                                   `json:"appTitle,omitempty"`
+	LoginLogo          *string                                   `json:"loginLogo,omitempty"`
+	MenuLogo           *string                                   `json:"menuLogo,omitempty"`
+	LoginBackground    *string                                   `json:"loginBackground,omitempty"`
+	LoginSubtitle      *string                                   `json:"loginSubtitle,omitempty"`
+	LoginBoxBackground *string                                   `json:"loginBoxBackground,omitempty"`
+	LoadingLogo        *string                                   `json:"loadingLogo,omitempty"`
+	HideEdition        *bool                                     `json:"hideEdition,omitempty"`
+	PublicDashboard    *FrontendSettingsPublicDashboardConfigDTO `json:"publicDashboard,omitempty"` // PR TODO: type this properly
```
Comment (AgnesToulet) [inline pkg/api/dtos/frontend_settings.go:116]: > // PR TODO: type this properly

What does this mean?

File: docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md [hunk 2]
```diff
@@ -120,7 +120,7 @@ To customize the footer of a public dashboard, add the following section to the
 ```ini
 [white_labeling.public_dashboards]
 
-# Hides the footer for the public dashboards if set to "true". If
+# Hides the footer for the public dashboards if set to `true`. If
```
Comment (AgnesToulet) [inline docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md:123]:
```suggestion
# Hides the footer for the public dashboards if set to `true`.
```

**Review by AgnesToulet — APPROVED** [review]
LGTM! 👍

File: docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md (hunk 2 above)
Comment (juanicabanas) [reply docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md:123]: good catch!

**Pull request merged**